### PR TITLE
Focus is on the first item in the dropdown

### PR DESCRIPTION
**Pull request by Steph4104**
Add focus to item in dropdown so screen reader will read the first line when the dropdown is open

#919

### CODE REVIEW

**Comment by Steph4104** [comment]
When I use the tab on notification, I got this. @ethanWallace 

image

**Comment by Steph4104** [comment]
@ethanWallace I add a new focus. Can you test with a screen reader if it works ?

File: mod/wet4/views/default/ajax/notif_dd.php [hunk 1]
```diff
@@ -7,7 +7,14 @@
 
 @author: Nick
 */
+?>
+<script>
+$( "li.elgg-menu-item-notifications div.message-dd-holder div.message-dd-block a" ).first().focus(); //add focus on the first link of notifications
+
+$( "li.elgg-menu-item-messages div.message-dd-holder div.message-dd-block a" ).first().focus(); //add focus on the first link of messages
```
Comment (nschonni) [inline mod/wet4/views/default/ajax/notif_dd.php]: Setting a second focus will make the previous focus unnecessary. Which should be given focus?

File: mod/wet4/views/default/page/elements/message_dd_list.php [hunk 2]
```diff
@@ -20,7 +20,7 @@
 		echo $no_results();
 		return;
 	}
-	echo "<p class='elgg-no-results message-dd-no-results'>$no_results</p>";
+	echo "<p class='elgg-no-results message-dd-no-results' aria-live='assertive'>$no_results</p>";
```
Comment (nschonni) [inline mod/wet4/views/default/page/elements/message_dd_list.php]: `aria-live` is to indicate that there is dynamic content that will change. This doesn't make sense for a paragraph with static results

Comment (Steph4104) [inline mod/wet4/views/default/page/elements/message_dd_list.php]: This paragraphe is in a dropdown menu generate by a ajax call. If aria-live is not there, the screen reader does not read the paragraphe.

File: mod/wet4/views/default/ajax/notif_dd.php (hunk 1 above)
Comment (Steph4104) [inline mod/wet4/views/default/ajax/notif_dd.php]: This focus is on a element from the dropdown in a ajax call. Only one focus is use at the time.

**Pull request merged**